packages/flutter_localizations/test/basics_test.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
```

Comment:
Do you need to add these tests to `dev/bots/test.dart`?

Reply:
Yes! Thanks for that.